pkg/image/build.go

```diff
@@ -360,3 +365,22 @@ func restoreDockerignore() error {
 
 	return os.Rename(dockerignoreBackupPath, ".dockerignore")
 }
+
+func ping() {
+	cli, err := client.NewClientWithOpts(client.FromEnv)
+	if err != nil {
+		fmt.Println("Failed to create docker client.")
+		return
+	}
+
+	ctxTimeout, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+
+	_, err = cli.Ping(ctxTimeout)
+
+	if err != nil {
+		fmt.Println("Failed to ping docker, please try restarting the docker daemon.")
```

Comment:
Minor nitpick, but I'd prefer to have this method return an error and have the call site responsible for communicating with the user.